### PR TITLE
chore: Upgrade to prettier 2.0

### LINKED ISSUE #7096: Upgrade to prettier 2.0
We should upgrade to latest prettier and revisit TSLint rulees along the way

### PR DESCRIPTION
**Pull request by dunqan**
Closes  #7096

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -166,7 +166,7 @@
     "karma-junit-reporter": "^1.2.0",
     "ng-packagr": "^9.0.0",
     "npm-package-versions": "^1.0.1",
-    "prettier": "1.19.1",
+    "prettier": "2.0.2",
```
Comment (tobi-or-not-tobi) [inline package.json]: I'd prefer adding the major with caret, and leave the patch (at least) up to the lock file. 

`"prettier": "^2.0.0",`

File: .prettierrc [hunk 2]
```diff
@@ -2,12 +2,10 @@
   "printWidth": 80,
   "tabWidth": 2,
   "singleQuote": true,
-  "trailingComma": "es5",
   "bracketSpacing": true,
   "jsxBracketSameLine": false,
```
Comment (tobi-or-not-tobi) [inline .prettierrc]: shell we get ride of `jsxBracketSameLine` as well?

Comment (tobi-or-not-tobi) [reply .prettierrc]: what about going back to a clean configuration that only requires non-defaults? 

```
{
  "singleQuote": true,
}
```

Or you prefer keeping the options as doc? I see people often going into our source an copying the whole thing, it helps to see the only opinionated thing is the singleQuote.

Comment (tobi-or-not-tobi) [reply .prettierrc]: I vote for cleaning it up.

**Review by tobi-or-not-tobi — CHANGES_REQUESTED** [review]
few minor suggestions/questions only

File: .prettierrc (hunk 2 above)
Comment (dunqan) [reply .prettierrc]: 👍

File: package.json (hunk 1 above)
Comment (dunqan) [reply package.json]: 👍

**Review by marlass — APPROVED** [review]

**Review by tobi-or-not-tobi — APPROVED** [review]

**Pull request merged**